ast/parser/EarthLexer.g4

```diff
@@ -131,6 +135,8 @@ COMMAND_B: COMMAND -> type(COMMAND), pushMode(COMMAND_ARGS);
 IMPORT_B: IMPORT -> type(IMPORT), pushMode(COMMAND_ARGS);
 CACHE_B: CACHE -> type(CACHE), pushMode(COMMAND_ARGS);
 HOST_B: HOST -> type(HOST), pushMode(COMMAND_ARGS);
+PIPELINE_B: PIPELINE -> type(PIPELINE), pushMode(COMMAND_ARGS);
+TRIGGER_B: TRIGGER -> type(TRIGGER), pushMode(COMMAND_ARGS);
```

Comment:
I think that we don't need these in the `BLOCK` mode. The `BLOCK` mode is only used for the bodies of `IF`, `FOR`, `WAIT` and `WITH DOCKER`.

Then again, it might help with providing a better error message if this check is done in the interpreter instead. Just thinking out loud.

Reply:
Sounds reasonable to leave it out for now.